src/main/extension/generated_extension_loader.cpp.in

```diff
@@ -9,10 +9,17 @@ ${EXT_LOADER_BODY}
     return false;
 }
 
-vector<string> linked_extensions = {${EXT_NAME_VECTOR_INITIALIZER}
-};
+const vector<string>& LinkedExtensions(){
+    static vector<string> VEC = {${EXT_NAME_VECTOR_INITIALIZER}
```

Comment:
Either the tidy check or the formatter is not going to be happy with VEC as variable name, `vec` is accepted

Reply:
The formatter pipeline/action complained about `vec` when I was running it. Same with the REGEX variable. 

Or I was doing something wrong.

Would happily change it to lowercase.

Reply:
I am surprised clang-tidy didn't complain about the global statics but the local ones... Feels flipped.